bin/oci-kvm

```diff
@@ -107,8 +107,8 @@ def parse_args():
                                        help='the name of the network')
     create_network_parser.add_argument('-v', '--vcn-cidr', action='store', required=True, type=str,
                                        help='The VCN CIDR block used by the choosen vNIC')
-    create_network_parser.add_argument('-F', '--ip-first', action='store', required=True, type=str,
-                                       help='first IP in virtual network address space')
+    create_network_parser.add_argument('-F', '--ip-bridge', action='store', required=True, type=str,
```

Comment:
Can we change the -F to -B to be consistent?

Reply:
Indeed. I was about too . part of my next commit